src/containers/userDirectory/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { Alert, Spin, Typography } from 'antd';
+import React, { useEffect, useState } from 'react';
+import { Helmet } from 'react-helmet';
+import protectedApiClient from '../../api/protectedApiClient';
+import { ChungusContentContainer } from '../../components';
+import UserInfoTable, { UserSummary } from '../../components/UserInfoTable';
+import {
+  AsyncRequest,
+  AsyncRequestCompleted,
+  AsyncRequestFailed,
+  asyncRequestIsComplete,
+  asyncRequestIsFailed,
+  asyncRequestIsLoading,
+  asyncRequestIsNotStarted,
+  AsyncRequestLoading,
+  AsyncRequestNotStarted,
+} from '../../utils/asyncRequest';
+
+const { Title } = Typography;
+
+const UserDirectory: React.FC = () => {
+  const [users, setContacts] = useState<AsyncRequest<UserSummary[], any>>(
+    AsyncRequestNotStarted(),
+  );
```

Comment:
👍

Reply:
Although, shouldn't we have our variable named contacts instead of users? Seems like the traditional way to do it.

Reply:
Or setUsers instead of setContacts. That makes more sense to me than users -> contacts